erpnext/quality_management/doctype/quality_procedure/test_quality_procedure.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018, Frappe and Contributors
+# See license.txt
+from __future__ import unicode_literals
+
+import frappe
+import unittest
+
+class TestQualityProcedure(unittest.TestCase):
+	def test_quality_procedure(self):
+		test_create_procedure = create_procedure()
+		test_get_procedure = get_procedure()
+		self.assertEquals(test_create_procedure, test_get_procedure)
```

Comment:
write a more meaningful test. make a nested procedure and test if `parent_quality_procedure` is set